Lib/_pycodecs.py

```
@@ -721,368 +967,522 @@
             p.append(ord(ch))
-    
-    #p += '\0'
+
+    # p += '\0'
     return p
 
-def charmapencode_output(c, mapping):
 
+def charmapencode_output(c, mapping):
     rep = mapping[c]
     if isinstance(rep, int) or isinstance(rep, int):
```

Pylint (3.3.7)

[refactor] Consider merging these isinstance calls to isinstance(rep, (int))

(R1701)